centreon/src/Core/HostTemplate/Infrastructure/Repository/DbWriteHostTemplateActionLogRepository.php

```diff
@@ -100,15 +100,15 @@ public function add(NewHostTemplate $hostTemplate): int
             if ($hostTemplateId === 0) {
                 throw new RepositoryException('Host template ID cannot be 0');
             }
-            
+
             $actionLog = new ActionLog(
-                self::HOST_TEMPLATE_OBJECT_TYPE,
+                ActionLog::OBJECT_TYPE_HOST,
```

Comment:
Maybe use ActionLog::OBJECT_TYPE_HOST_TEMPLATE instead.

Reply:
Was thinking about it, but in legacy Host Templates are referenced as "host" in action logs and I didn't want to change the original behaviour.

I could also create a OBJECT_TYPE_HOST_TEMPLATE constant that will have "host" as value so original behaviour will still not be changed and be less confusing while reading.